fenix/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -1438,4 +1420,28 @@
                 app:popUpToInclusive="true" />
         </fragment>
     </navigation>
+    <navigation
```

Comment:
nit: let's add a new line before this to separate out the `<navigation>`

Reply:
Done